src/TLuaInterpreter.cpp

```diff
@@ -80,6 +82,9 @@ int luaopen_yajl(lua_State*);
 
 using namespace std;
 
+QTextToSpeech* speechUnit;
```

Comment:
Ah ☕️ _(coffee)_ -  the ambrosia/life-blood of late night coders around the globe... 🤣